src/txreconciliation.cpp

```diff
@@ -154,6 +178,19 @@ class TxReconciliationTracker::Impl {
             full_salt.GetUint64(1), we_initiate, flood_to));
     }
 
+    void AddToReconSet(NodeId peer_id, const std::vector<uint256>& txs_to_reconcile)
+    {
+        LOCK(m_mutex);
+        auto recon_state = m_states.find(peer_id);
+        assert(recon_state != m_states.end());
+        for (auto& wtxid: txs_to_reconcile) {
+            recon_state->second.m_local_set.m_wtxids.insert(wtxid);
+        }
+
+        LogPrint(BCLog::NET, "Added %i transactions to the reconciliation set for peer=%d.\n",
```

Comment:
83a08c5ea07f7f7e3e67501ac8666521427ae9d3

Could `m_local_set already` have some of `txs_to_reconcile`? If so maybe we should count successful `.insert()` and log that?